ui/pages/accept-invite/index.js

```diff
@@ -18,6 +21,29 @@ export default function AcceptInvite() {
 
   return (
     <div className='flex w-full flex-col items-center px-10 pt-4 pb-6'>
+      <h1 className='text-base font-bold leading-snug'>Accept Invitation</h1>
+      {google && (
+        <>
+          <Providers
+            providers={[google]}
+            authnFunc={oidcLogin}
+            baseDomain={baseDomain}
+            loginDomain={loginDomain}
+            buttonPrompt={'Log in with'}
+          />
+          <div className='relative mt-6 mb-2 w-full'>
+            <div
+              className='absolute inset-0 flex items-center'
+              aria-hidden='true'
+            >
+              <div className='w-full border-t border-gray-200' />
+            </div>
+            <div className='relative flex justify-center text-sm'>
+              <span className='bg-white px-2 text-2xs text-gray-400'>OR</span>
+            </div>
+          </div>
+        </>
+      )}
       <PasswordResetForm
         header='Welcome to Infra'
```

Comment:
We should remove these props if these are unused. However, this will this affect the password reset flow experience, right? We may need to add an `h1` and `h2` back to the forgot password "reset" page

Reply:
Missed this on the reset form. It should be good now.